### ISSUE TITLE
Latest logbook-netty is not part of logbook-bom

### ISSUE THREAD

**Issue by gmariotti** [zalando/logbook#741]
## Description

When using the `logbook-bom` with version `2.1.0`, the `logbook-netty` version pulled is `2.0.0-SNAPSHOT` instead of the expected `2.1.0`.

## Possible Fix

Publish a fixed or new version of the bom with the correct version of `logbook-netty` https://github.com/zalando/logbook/blob/master/logbook-bom/pom.xml#L57

## Steps to Reproduce

Create a new project with the following `build.gradle.kts`

```kotlin
plugins {
  kotlin("jvm") version "1.3.71"
}

dependencies {
  implementation(platform("org.zalando:logbook-bom:2.1.0"))

  implementation("org.zalando:logbook-core")
  implementation("org.zalando:logbook-json")
  implementation("org.zalando:logbook-logstash")
  implementation("org.zalando:logbook-netty")
}
```

And then run `./gradlew dependencyInsight --dependency logbook-netty`

The output will be

```
org.zalando:logbook-netty:2.0.0-SNAPSHOT (by constraint) FAILED
   Failures:
      - Could not find org.zalando:logbook-netty:2.0.0-SNAPSHOT.
        Searched in the following locations:
          - https://dl.google.com/dl/android/maven2/org/zalando/logbook-netty/2.0.0-SNAPSHOT/maven-metadata.xml
          - https://dl.google.com/dl/android/maven2/org/zalando/logbook-netty/2.0.0-SNAPSHOT/logbook-netty-2.0.0-SNAPSHOT.pom
          - https://repo.maven.apache.org/maven2/org/zalando/logbook-netty/2.0.0-SNAPSHOT/maven-metadata.xml
          - https://repo.maven.apache.org/maven2/org/zalando/logbook-netty/2.0.0-SNAPSHOT/logbook-netty-2.0.0-SNAPSHOT.pom
          - https://jcenter.bintray.com/org/zalando/logbook-netty/2.0.0-SNAPSHOT/maven-metadata.xml
          - https://jcenter.bintray.com/org/zalando/logbook-netty/2.0.0-SNAPSHOT/logbook-netty-2.0.0-SNAPSHOT.pom

org.zalando:logbook-netty:2.0.0-SNAPSHOT FAILED
\--- org.zalando:logbook-bom:2.1.0
     \--- compileClasspath

org.zalando:logbook-netty -> 2.0.0-SNAPSHOT FAILED
\--- compileClasspath
```

## Your Environment

* Version used:
* Link to your project:

**Comment by whiskeysierra** [comment]
Would you mind creating a pull request for it?

**Comment by gmariotti** [comment]
Sure, should I just fix the dependency in the bom?

**Comment by whiskeysierra** [comment]
I'd say yes

**Comment by gmariotti** [comment]
Hi @whiskeysierra, I opened #742 but didn't get any feedback about it. Did you have a chance to give it a look?

**Comment by whiskeysierra** [comment]
Actually, now that I took another look. The proper fix would be to just bump the version of logbook-netty within the bom to the latest snapshot. No other changes needed. And then we release a new version.

**Comment by gmariotti** [comment]
Hi @whiskeysierra, I just wanted to let you know that this issue is still present in version 2.1.1.

**Comment by whiskeysierra** [comment]
Sorry, I missed that. I just created #789 with a fix for it.

**Comment by whiskeysierra** [comment]
I just released 2.1.2 with a fix.
